tests/conftest.py

```diff
@@ -0,0 +1,86 @@
+import subprocess
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Mapping
+
+import pytest
+import requests
+
+from tests.services import API_URL, EDITOAST_URL
+from tests.utils.timetable import create_op_study, create_project, create_scenario
+
+
+@dataclass(frozen=True)
+class Scenario:
+    project: int
+    op_study: int
+    scenario: int
+    infra: int
+    timetable: int
+
+
+def _load_generated_infra(name: str) -> int:
+    generator = Path(__file__).resolve().parents[1] / "core/examples/generated/generate.py"
+    output = Path("/tmp/osrd-generated-examples")
+    infra = output / f"{name}/infra.json"
+    subprocess.check_call(["python", str(generator), str(output), name])
```

Comment:
```suggestion
    subprocess.check_call([sys.executable, str(generator), str(output), name])
```

(with an added `sys` import)

I completely cleaned up my environment to test this (on windows), and I had an error where libraries were missing in the generator. Apparently calling `python` like this keeps the same env on linux, but on windows it calls the "default" python env with no library. This change fixes it.

Another solution would be to import the generator as a library directly, but it doesn't seem that straightforward here.